include/server/api/base_parameters_grammar.hpp

```diff
@@ -111,15 +111,15 @@ struct BaseParametersGrammar : boost::spirit::qi::grammar<Iterator, Signature>
                                                 qi::_1,
                                                 qi::_2)];
 
-        location_rule =
-            (double_ > qi::lit(',') >
-             double_)[qi::_val = ph::bind(
-                          [](double lon, double lat) {
-                              return util::Coordinate(util::toFixed(util::FloatLongitude{lon}),
-                                                      util::toFixed(util::FloatLatitude{lat}));
-                          },
-                          qi::_1,
-                          qi::_2)];
+        location_rule = (double_ > qi::lit(',') >
+                         double_)[qi::_val = ph::bind(
+                                      [](double lon, double lat) {
+                                          return util::Coordinate(
+                                              util::toFixed(util::UnsafeFloatLongitude{lon}),
```

Comment:
Here numeric_cast also not really needed, because it :
* checks overflows
* checks underflows
* makes rounding  accordingly to a rounding policy 

double to double conversion with default rounding policy would be similar to static_cast. 

One thing that is allowed atm  is [parsing of NaNs](https://github.com/boostorg/spirit/blob/abe2a14a7fd281e0e6a6a32ab3cde31f277fbb64/include/boost/spirit/home/qi/numeric/detail/real_impl.hpp#L207-L208)
but it can be rejected by providing `RealPolicy` that fail at `parse_nan` and `parse_inf`